packages/config/src/globals.ts

```diff
@@ -191,4 +191,12 @@ export const CONTROL_ALL = 'control-all:'
  * @param event - name of event that just happened
  */
 export const UNSENT_CHAT_MESSAGE_TYPE = 'chat'
-export const UNSENT_SELECT_BY_DEFAULT_VALUE = 'BY_DEFAULT_VALUE'
\ No newline at end of file
+export const UNSENT_SELECT_BY_DEFAULT_VALUE = 'BY_DEFAULT_VALUE'
+
+export const SUPPORT_PANEL_LAYOUT = {
```

Comment:
@lebaphi - could you change these constants by putting `serge.planning` at the front of the string?  

That may help us navigate the cookies in the future.

Reply:
sure @IanMayo, I will update in part 2 of this PR 😄